### DISCUSSION TITLE
identifying differentially spliced genes between two conditions

### DISCUSSION THREAD

**Discussion by Flu09** [pyrovelocity/pyrovelocity#597] [Q&A] [answered]
First, thank you for creating this tool. I have already finished generating the loom files from the filtered reads through running cellranger + velocyto . I have two conditions (disease and healthy samples) . I wonder if pyrovelocity could be helpful in what I want to find.  I am interested about finding the deferentially spliced genes in each cell type compared to controls and and i want to find the top spliced in each type in the disease state compared to controls.

**Comment by cameronraysmith** [comment, marked as answer]
@Flu09 we don't currently have a built-in function to perform differential expression analysis on unspliced count matrices; however, we may be able to illustrate how to do this in the documentation.

In the meantime, you may be able to do this using scanpy, which is installed as a dependency of pyrovelocity or can be installed separately. You might find [this tutorial](https://nbisweden.github.io/workshop-scRNAseq/labs/scanpy/scanpy_05_dge.html) or another similar one helpful to review.

**Comment by Flu09** [comment]
Thank you so much. I want to ask which counts should be used the spliced or the unspliced? You probably meant to replace the normal counts with the matrix of one of them right? then find DEGs?

**Reply by cameronraysmith** [reply]
> I am interested about finding the deferentially spliced genes in each cell type compared to controls and and i want to find the top spliced in each type in the disease state compared to controls.

Depending on what you would like to evaluate when you refer to "differentially spliced genes" you may indeed want to attempt to use all available count matrices noting the limitations given their relative sparsity.
